Automattic/wp-calypso
# Add placeholder Pricing Upgrade block

#### Proposed Changes

This Pull Request adds the basic structure and scripts for a new Pricing Upgrade custom Gutenberg Block.

At this stage, the block is just a dummy placeholder on top of which we'll iterate to implement the feature.

#### Testing Instructions



1. Run `yarn test` and ensure tests are working
2. Run `yarn build` and ensure production builds are working
3. After merging and deploying, the plugin should be possible to install as described [here](https://github.com/Automattic/wp-calypso/pull/69324/files#diff-1640d493be4b0646c317f3d0c2d644819efaea729005cbfde803e632d9600a8bR25-R33)

#### Pre-merge Checklist






Related to #
